osxkeychain.go

```diff
@@ -176,43 +188,61 @@ func AddInternetPassword(pass *InternetPassword) error {
 //
 // Returns an error if the lookup was unsuccessful.
 func FindInternetPassword(pass *InternetPassword) (*InternetPassword, error) {
-	resp := InternetPassword{}
-	protocol := protocolTypeToC(pass.Protocol)
-	authtype := C.SecAuthenticationType(C.kSecAuthenticationTypeHTTPBasic)
-	var cpassword unsafe.Pointer
-	var cpasslen C.UInt32
+	// TODO: Check fields for UTF-8 encoding and size fitting in
+	// 32 bits.
```

Comment:
As we discussed on a3c7aea, it should be pretty easy to just do this validation where it's needed rather than leaving comments for it. Thanks